<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find joy_control)/config/params.yaml" />

  <node respawn="true" pkg="joy" type="joy_node" name="ps4_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
  </node>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baud" value="57600" />
  </node>

  <node pkg="dynamixel_sdk_examples" type="read_write_node" name="read_write_node">
  </node>

  <node pkg="joy_control" type="joy_control" name="joy_control" output="screen">
  </node>
</launch>
